src/Analyzer.TemplateProcessor/ArmTemplateProcessor.cs

```diff
@@ -193,7 +193,8 @@ internal Template ProcessResourcesAndOutputs(Template template)
                     }
                     catch (Exception)
                     {
-                        logger?.LogWarning("The parsing of a template output failed. Output name: {outputName}. Output value: {outputValue}", outputKey, template.Outputs[outputKey]?.Value?.Value?.ToString());
+                        logger?.LogWarning("The parsing of a template output failed");
```

Comment:
Here too should there be a link?

Reply:
A link to what?

Reply:
the path to the template in the warning

Reply:
Templates have only one output object and we don't have information about what part of the object failed

Reply:
Maybe you can include just the outputKey in the message still.